library/foundations/scripts/mapping/css-generator.ts

```diff
@@ -28,8 +28,8 @@ export default {
       /--other.*-direction/,
       // replace `vertical` with `column` and `horizontal` with `row`
       [
-        [/vertical/g, 'column'],
-        [/horizontal/g, 'row'],
+        [/^vertical$/, 'column'],
+        [/^horizontal$/, 'row'],
```

Comment:
This was replacing the target words in full token reference strings - was a very confusing problem until I figured out what was happening